src/gennylib/include/gennylib/v1/Sleeper.hpp

```diff
@@ -49,6 +50,18 @@ class Sleeper {
     Sleeper(Sleeper&& other) = delete;
     Sleeper& operator=(Sleeper&& other) = delete;
 
+    constexpr void sleep_for(Orchestrator& o, const PhaseNumber pn,
```

Comment:
💅🏽 on the spacing (run clang-format) and `camelCase` conventions
And don't hate me, but `o` and `pn` were hard to parse in the admittedly simple impl. Although I just realized you're copying the existing names. Shrug. Your call on the param names.